src/operator/subgraph/build_subgraph.cc

```diff
@@ -561,10 +561,12 @@ void CutGraphInputs(const std::vector<nnvm::NodeEntry*> &input_entries,
     nnvm::ObjectPtr n = nnvm::CreateVariableNode(
         var_name + std::to_string(name_count_map[var_name]));
     // set attribute for subgraph input to indicate if it is from an arg/param to model
-    if (e->node->is_variable())
+    if (e->node->is_variable()) {
       n->attrs.dict["isArg"] = "True";
-    else
+      n->attrs.dict["argName"] = var_name;
```

Comment:
sanity check .... are we attaching "auxName" to auxes ?

Reply:
no. argName will be set for all "variables" which includes both args and aux. for an aux param, isArg = True && isAux = true && argName = 'bob'

Reply:
Heres an example from resnet's batchNorm operator:
```
{
      "op": "null",
      "name": "resnetv10_batchnorm0_beta0",
      "attrs": {
        "argName": "resnetv10_batchnorm0_beta",
        "isArg": "True",
        "isAux": "False"
      },
      "inputs": []
    },
    {
      "op": "null",
      "name": "resnetv10_batchnorm0_running_mean0",
      "attrs": {
        "argName": "resnetv10_batchnorm0_running_mean",
        "isArg": "True",
        "isAux": "True"
      },
      "inputs": []
    },
```

Reply:
Heres an example of an input to a subgraph thats an output from another operator (ie. not a param):
```
{
      "op": "null", 
      "name": "_plus0_output0", 
      "attrs": {
        "isArg": "False", 
        "isAux": "False"
      }, 
      "inputs": []
    }, 
```